lightning/src/ln/monitor_tests.rs

```diff
@@ -398,13 +430,37 @@ fn do_test_claim_value_force_close(prev_commitment_tx: bool) {
 	mine_transaction(&nodes[0], &remote_txn[0]);
 	mine_transaction(&nodes[1], &remote_txn[0]);
 
-	let b_broadcast_txn = nodes[1].tx_broadcaster.txn_broadcasted.lock().unwrap().split_off(0);
+	if anchors {
+		let mut events = nodes[1].chain_monitor.chain_monitor.get_and_clear_pending_events();
+		assert_eq!(events.len(), 1);
+		match events.pop().unwrap() {
+			Event::BumpTransaction(bump_event) => {
+				let mut first_htlc_event = bump_event.clone();
+				if let BumpTransactionEvent::HTLCResolution { ref mut htlc_descriptors, .. } = &mut first_htlc_event {
+					htlc_descriptors.remove(1);
+				} else {
+					panic!("Unexpected event");
+				}
+				let mut second_htlc_event = bump_event;
+				if let BumpTransactionEvent::HTLCResolution { ref mut htlc_descriptors, .. } = &mut second_htlc_event {
+					htlc_descriptors.remove(0);
```

Comment:
Not sure if I understand this. Why can't we do this in the previous block?

Reply:
Oh, ok. I see now how they are passed individually below. Unclear to me why `bump_event` can't be passed directly instead, though.